recipe/meta.yaml

```diff
@@ -14,6 +14,13 @@
 {% set cross_target_platform = "linux-64" %}
 {% endif %}
 
+{% if cross_target_platform.startswith("win-") %}
+{% set runtime_suffix = "" %}
+{% else %}
+{% set runtime_suffix = "-ng" %}
```

Comment:
Longterm it would be nice to get rid of the runtime suffix everywhere; the last respective linux builds are:
* [libgcc](https://anaconda.org/conda-forge/libgcc/files): 5.2.0 (Aug. 2018)
* [libgfortran](https://anaconda.org/conda-forge/libgfortran/files?page=3): 3.0.0 (Aug. 2018)
* [libstdcxx](https://anaconda.org/conda-forge/libstdcxx/files): never published for linux

This should be fixable with some simple run-constraints (plus keeping `-ng` as a wrapper until we've switched over the strong run-exports for maybe a few years).